package.json

```diff
@@ -3,6 +3,7 @@
   "license": "Unlicense",
   "version": "7.5.0-beta.2",
   "dependencies": {
+    "@joepie91/v4l2camera": "^1.0.5",
```

Comment:
On line `49`, there's the same dependency, but on another version and possibly repo. Check with Rafael whether that optionalDependency should be removed or not and if it is worth for us to create our own fork of the v412camera repository